config/locales/server.en.yml

```diff
@@ -554,6 +554,10 @@ en:
 
       Are you sure you want to continue this old conversation?
 
+    user_not_seen:
+      single: "The person you are messaging, %{username}, hasn’t been seen here in a very long time – %{time_ago}. They may not receive your message. You may wish to seek out alternate methods of contacting %{username}."
```

Comment:
```suggestion
      single: "The user @%{username}, hasn’t been seen here in a long time – %{time_ago}. They may not receive your message. You may wish to seek out alternate methods of contacting them."
```

Can we also try to make the username linkable, maybe via `<a class="mention" href="/u/username">@username</a>`?

Reply:
Done.

![SCR-20220927-g6v](https://user-images.githubusercontent.com/5732281/192446388-ce573680-cd3f-45f7-9cec-c616fb638e93.png)